src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -757,6 +825,12 @@ private void SaveTokens(AuthenticationProperties properties, OpenIdConnectMessag
                 }
             }
 
+            var sid = principal.FindFirst("sid")?.Value;
+            if (!string.IsNullOrEmpty(sid))
+            {
+                tokens.Add(new AuthenticationToken { Name = "sid", Value = sid });
```

Comment:
No particular reason. It was mainly for consistency with how we extract `id_token_hint`, but I'll revert it if you prefer.